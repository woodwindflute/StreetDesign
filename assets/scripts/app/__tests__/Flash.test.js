/* eslint-env jest */
import React from 'react'
import { render } from '../../../../test/helpers/render'
import Flash from '../Flash'

describe('Flash', () => {
  it('renders', () => {
    const { asFragment } = render(<Flash />)
    expect(asFragment()).toMatchSnapshot()
  })
})
