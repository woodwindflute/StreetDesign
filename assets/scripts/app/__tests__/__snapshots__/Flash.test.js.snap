// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flash renders 1`] = `
<DocumentFragment>
  <div
    class="flash"
  />
</DocumentFragment>
`;
